server/init.go

```diff
@@ -22,7 +21,7 @@ func GenerateCoinKey() (sdk.AccAddress, string, error) {
 
 // GenerateSaveCoinKey returns the address of a public key, along with the secret
 // phrase to recover the private key.
-func GenerateSaveCoinKey(keybase keyring.Keyring, keyName, keyPass string, overwrite bool) (sdk.AccAddress, string, error) {
```

Comment:
`keyPass` wasn't being used

Reply:
Yes, and thanks for cleaning this up. It was a leftover from the old keybase -> keyring migration